package.json

```diff
@@ -25,9 +25,13 @@
     "pkg": "run-s clean format build:server && pkg --out-path bin .pkg.config.json",
     "test": "run-s build:server && node -r esm ./node_modules/.bin/mocha 'test/**/*.js' --recursive"
   },
+  "engines": {
+    "node": ">=14 <15",
+    "npm": ">=7"
+  },
```

Comment:
@soedirgo only tried npm v7 and node v14 locally but i'm sure it'll work with a few other major versions so feel free to expand the ranges. Gonna save the next person a lot of time (worst case they get a warning) when queries hang and they have no clue why it's happening only to realize it had to do with their node version 😭